views/outbound-faxes-rows.html

```diff
@@ -22,9 +22,9 @@
 			<div class="top-line">{{toFriendlyDate created 'date'}}</div>
 			<div class="bottom-line">{{toFriendlyDate created 'time'}}</div>
 		</td>
-		<td class="select-line">{{formatted.sendingFaxbox}}</td>
-		<td class="select-line">{{#monsterNumberWrapper formatted.sendingNumber}}{{/monsterNumberWrapper}}</td>
-		<td class="select-line">{{#monsterNumberWrapper formatted.receivingNumber}}{{/monsterNumberWrapper}}</td>
+		<td class="select-line" data-sort-value="{{formatted.sendingFaxbox}}">{{formatted.sendingFaxbox}}</td>
```

Comment:
The only possible type for `formatted.sendingFaxbox` is `string` so it will be parsed as text by default.
```suggestion
		<td class="select-line">{{formatted.sendingFaxbox}}</td>
```